### DISCUSSION TITLE
Fuego Roadmap & Sponsorship

### DISCUSSION THREAD

**Discussion by EwenQuim** [go-fuego/fuego#136] [General]
Hello everyone! 🔥 

After a long month's absence, I've finally got some time to work on the Fuego project. I've got some good news for you: Fuego is going to be sponsored by Zuplo! This will free me up for one day a month, and we'll be able to use the money for servers or Continuous Integration time. I'd also like to give some of the money back to the most active contributors (@dylanhitt @rizerkrof) . Don't hesitate to contact me via the discord ([link here](https://discord.com/channels/118456055842734083/1212766266075390004) [invite here](https://discord.com/invite/s233GcZqFT)).

As for the roadmap, there are several possible avenues.

- Fix the bugs identified, of course
- Header-dependent return typing `Accept` (draft in progress by @EwenQuim) (to be able to return HTML or JSON with the same route)
- OpenAPI 3.1 support
- Header/QueryParam typing

Feel free to take topics if you'd like to see Fuego evolve!

**Comment by dylanhitt** [comment]


**Reply by EwenQuim** [reply]
https://discord.com/invite/s233GcZqFT

**Comment by asahasrabuddhe** [comment]
I would like to help contribute to Fuego's future. Where do we usually discuss the direction of development? GitHub or Discord?

**Reply by EwenQuim** [reply]
Hi @asahasrabuddhe and thank you for contributing the last few days. It's thanks to the contributions that the project grows and improves!

We usually do everything on Github

- propositions/questions on Discussion
- bugs on Issues

What you can do now is review the #176 spec and then, its associated PR. It's a huge improvement for Fuego and it takes some time to review, so good luck!

We'll also have a few things to work on top of this PR, like:
- making the `param.Required()` trigger an error if the parameter is now present in the request.
- include `option.Path` option in the routes
